packages/nimble-components/src/rich-text/viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -154,30 +160,22 @@ describe('RichTextViewer', () => {
             ).toEqual('1');
         });
 
-        // TODO: Once the rich text validator (https://github.com/ni/nimble/pull/1688) added for duplicate configuration elements, below test case should be updated
-        it('adding two mention configuration elements in the same viewer should render as mention view', async () => {
+        it('adding duplicate mention configuration elements in the same viewer should not render as mention view', async () => {
             element.markdown = '<user:1>';
             await appendUserMentionConfiguration(element);
             await appendUserMentionConfiguration(element);
 
             expect(pageObject.getRenderedMarkdownTagNames()).toEqual([
                 'P',
-                `${richTextMentionUsersViewTag}`.toUpperCase()
+                'NIMBLE-ANCHOR'
```

Comment:
Is there a reason not to use `anchorTag`?

Reply:
No, it should be `anchorTag`. I have updated the same